### ISSUE TITLE
[BUG] silverstripe-cache dir is missing

### ISSUE THREAD

**Issue by a2nt** [lekoala/silverstripe-debugbar#103]
```
[Emergency] Uncaught UnexpectedValueException: DirectoryIterator::__construct(/srv/dist/web/silverstripe-cache/www-data/debugbar/): failed to open dir: No such file or directory
GET /?flushtoken=4bcb6cc6cb95cdc471d385737ce2aca5&flush=
Line 113 in /srv/dist/web/vendor/maximebf/debugbar/src/DebugBar/Storage/FileStorage.php
```

**Comment by robbieaverill** [comment]
Can you please provide the versions of SilverStripe and debugbar you're using?

**Comment by robbieaverill** [comment]
Feel free to reopen this issue with some supporting information =) Thanks for reporting!

**Comment by JakubDolba** [comment]
same error just happened to me, I believe it is cause by missing tmp directory and used \DIrectoryIterator in FileStorage implementation

- SS 4.2
- executing "flush" right after composer require 

see #110

**Comment by Leapfrognz** [comment]
I am also getting this error.

SS 4.3.1
Executing flush from front end. Dont get it while running via CLI.
```
[Emergency] Uncaught UnexpectedValueException: DirectoryIterator::__construct(/var/tmp/ ... /daemon/debugbar/): failed to open dir: No such file or directory
111
```
This cache directory doesnt exist.

**Comment by robbieaverill** [comment]
Are you all using Vagrant or some other kind of virtualisation system?

**Comment by robbieaverill** [comment]
I've merged #110 which should prevent the exception from occurring

**Comment by Leapfrognz** [comment]
Thanks, Can you tag a release?

**Comment by robbieaverill** [comment]
Sure - see https://github.com/lekoala/silverstripe-debugbar/releases/tag/2.0.2

I'll close this as fixed now, reopen if still happening with 2.0.2

**Comment by Leapfrognz** [comment]
Thanks :)
